al2022neu.pkr.hcl

```diff
@@ -0,0 +1,34 @@
+locals {
+  ami_name_al2022neu = "${var.ami_name_prefix_al2022}-neu-hvm-2022.0.${var.ami_version}-x86_64-ebs"
```

Comment:
Not sure but is there any particular reason to keep the platform type 3 letters? Wondering if maybe we should call it `neuron-hvm-2022...` for the external name

Reply:
I don't have specific reason for this and I prefer full name too 😂 Just following the convention